src/tests/electron/common/view-controllers/app-controller.ts

```diff
@@ -45,11 +45,19 @@ export class AppController {
         const automatedChecksView = new AutomatedChecksViewController(this.client);
         await automatedChecksView.waitForViewVisible();
 
-        this.setToMinimumSupportedWindowSize();
```

Comment:
I think the original name was more clear; consider keeping this name for the outer function and instead renaming (or inlining) the inner function